src/Microsoft.DotNet.Build.Tasks/PackageFiles/buildagainstpackages.targets

```diff
@@ -182,6 +182,7 @@
                                   VersionStructureRegex="$(VersionStructureRegex)"
                                   AdditionalDependencies="@(_InjectProjectReferenceDependency)"
                                   ProjectJson="$(OriginalProjectJson)"
+                                  IdentityRegex="$(BuildTestsAgainstPackagesIdentityRegex)"
```

Comment:
I have an additional PR for the corefx repo which defines this property (regex).  I'll submit that PR after this PR is reviewed / merged.